regression/scripts/Vehicles/Stopped_Rotor_CRM.py

```diff
@@ -0,0 +1,542 @@
+from SUAVE.Methods.Propulsion.electric_motor_sizing import size_from_mass , compute_optimal_motor_parameters
+from SUAVE.Methods.Propulsion import propeller_design   
+from SUAVE.Methods.Weights.Buildups.Electric_Lift_Cruise.empty import empty
+
+import numpy as np
+import pylab as plt
+from copy import deepcopy 
+
+# ----------------------------------------------------------------------
+#   Build the Vehicle
+# ----------------------------------------------------------------------
+def sr_vehicle_setup():
+    
+    # ------------------------------------------------------------------
+    #   Initialize the Vehicle
+    # ------------------------------------------------------------------    
+    vehicle               = SUAVE.Vehicle()
+    vehicle.tag           = 'Lift_Cruise_CRM'
+    vehicle.configuration = 'eVTOL'
+    # ------------------------------------------------------------------
+    #   Vehicle-level Properties
+    # ------------------------------------------------------------------    
+    # mass properties
+    vehicle.mass_properties.takeoff           = 2450. * Units.lb 
+    vehicle.mass_properties.operating_empty   = 2250. * Units.lb               # Approximate
+    vehicle.mass_properties.max_takeoff       = 2450. * Units.lb               # Approximate
+    vehicle.mass_properties.center_of_gravity = [2.0144,   0.  ,  0. ] # Approximate
+    
+    # basic parameters 
+    vehicle.reference_area                    = 10.76 	
+    vehicle.envelope.ultimate_load            = 5.7   
+    vehicle.envelope.limit_load               = 3.  
+    
+    # ------------------------------------------------------				
+    # WINGS				
+    # ------------------------------------------------------				
+    # WING PROPERTIES	
+    wing = SUAVE.Components.Wings.Main_Wing()
+    wing.tag	                  = 'main_wing'		
+    wing.aspect_ratio	          = 10.76 
+    wing.sweeps.quarter_chord	  = 0.0	 * Units.degrees
+    wing.thickness_to_chord	  = 0.18		
+    wing.taper	                  = 1. 	
+    wing.span_efficiency	  = 0.9		
+    wing.spans.projected	  = 35.0   * Units.feet
+    wing.chords.root	          = 3.25   * Units.feet
+    wing.total_length	          = 3.25   * Units.feet	
+    wing.chords.tip	          = 3.25   * Units.feet	
+    wing.chords.mean_aerodynamic  = 3.25   * Units.feet		
+    wing.dihedral	          = 1.0    * Units.degrees		
+    wing.areas.reference	  = 113.75 * Units.feet**2	
+    wing.areas.wetted	          = 227.5  * Units.feet**2		
+    wing.areas.exposed	          = 227.5  * Units.feet**2		
+    wing.twists.root	          = 4.0    * Units.degrees		
+    wing.twists.tip	          = 0.0    * Units.degrees			
+    wing.origin	                  = [1.5, 0., 0. ] 
+    wing.aerodynamic_center	  = [1.975 , 0., 0.]    
+    wing.winglet_fraction         = 0.0  
+    wing.symmetric                = True
+    wing.vertical                 = False
+    
+    # Segment 	
+    segment = SUAVE.Components.Wings.Segment()
```

Comment:
Clean these data blocks a bit (table format)

Reply:
Scripts changed to table format